### PR TITLE
Publicize: release to proper plans

### PR DESCRIPTION
**Pull request by artpi**
## This releases the feature. Don't merge before we decide to release.

This accomplishes several things (I know thats bad, but splitting that would need interim approaches)
- Drop support for JP personal / wpcom personal
- Remove "scheduling" feature
- Repurpose "republicize" feature as "advanced social media"
- Update plans page

Merging this PR will open the feature to wpcom users.

__Yes, we need to rename internal use of `republicize` to `advanced-social-media`, but that is a task for another PR__

### wpcom plans page 

zrzut ekranu 2017-06-21 o 16 32 35

### Jetpack plans page
zrzut ekranu 2017-06-21 o 16 40 02

### wpcom free, wpcom personal

zrzut ekranu 2017-06-21 o 16 33 37

### Jetpack free, jetpack personal

zrzut ekranu 2017-06-21 o 16 34 30

### Jetpack premium, wpcom premium, JP business, wpcom business

zrzut ekranu 2017-06-21 o 16 35 42

## Testing instructions

Make sure both instant "share" and scheduling works only on:
- wpcom premium
- wpcom business
- jp premium
- jp business

and that that everything works fine

### CODE REVIEW

**Comment by clucasrowlands** [comment]
A few copy suggestions in the screenshots above:

1) Popup copy on wpcom plans page:

> Plan your social media activity in
> advance and schedule the updates
> to be posted at a perfect time.

I'd suggest changing "activity" and "a perfect time." Some ideas:

> Plan your social media messages
> in advance and schedule updates
> to publish in the future.

> Draft social media messaging
> in advance and schedule updates
> at any time in the future.

> Schedule your social media 
> updates in advance and promote
> your posts when it's best for you.

---

2) Copy on wpcom free, wpcom personal:

> Easy monetization options

I suggest avoiding "easy" and maybe using copy we've previously used on the plans page. Suggestions (don't forget the final periods):

- Monetization options.
- Monetize your site.

---

3) Copy on Jetpack free, jetpack personal

> Easy monetization options 

Same suggestions as in #2 above.

- Monetization options.
- Monetize your site.

> Daily Malware Scanning

Be consistent (use sentence case and insert final period):

> Daily malware scanning.

Finally, in the blue button copy, it looks like there's a stray "A" before "$".

I think that's it—but let me know if I missed other copy that needs to be reviewed. Thanks!

File: client/blocks/post-share/publicize-actions-list.jsx [hunk 1]
```diff
@@ -192,14 +189,11 @@ class PublicizeActionsList extends PureComponent {
...
 
-		if ( hasRepublicizeFeature && ! hasRepublicizeSchedulingFeature ) {
-			return <UpgradeToPremiumNudge { ...this.props } />;
-		}
 		if ( scheduledActions.length === 0 ) {
 			return <Notice
 				status="is-info"
 				showDismiss={ false }
-				text={ translate( 'Did you know you can decide exactly when Publicize shares your post? You can!' +
+				text={ translate( 'Did you know you can decide exactly when Publicize shares your post? You can! ' +
```
Comment (a8ci18n) [inline client/blocks/post-share/publicize-actions-list.jsx]: Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Did you know you can decide exactly when Publicize shares your post? You can!Click the calendar icon next to "Share post" to schedule your social shares.' )` *ES Score: 18*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/lib/plans/constants.js [hunk 2]
```diff
@@ -574,17 +568,9 @@ export const FEATURES_LIST = {
...
 		getSlug: () => FEATURE_REPUBLICIZE,
-		getTitle: () => i18n.translate( 'Targeted Social Media' ),
-		getDescription: () => i18n.translate(
-			'Share your posts as many times as you like on all your connected social media accounts using Publicize.'
-		)
-	},
-	[ FEATURE_REPUBLICIZE_SCHEDULING ]: {
-		getSlug: () => FEATURE_REPUBLICIZE_SCHEDULING,
-		getTitle: () => i18n.translate( 'Targeted Social Media' ),
+		getTitle: () => i18n.translate( 'Advanced Social Media' ),
```
Comment (a8ci18n) [inline client/lib/plans/constants.js]: :disappointed: 32 existing translations will be lost with this change.

**Comment by artpi** [comment]
> I'd suggest changing "activity" and "a perfect time." Some ideas:

Thanks @clucasrowlands !
I changed it to the last suggestion :)

Regarding Monetization options and other features this is not touching - I didnt change them since these are totally unrelated and out of scope of this PR

**Pull request merged**